src/workspacer/Plugins/PluginManager.cs

```diff
@@ -52,8 +50,7 @@ private static Type GetPluginFromFolder(string dir)
             var name = new DirectoryInfo(dir).Name;
 
             Console.WriteLine(Path.Combine(dir, name + ".dll"));
-            var assembly = Assembly.LoadFile(Path.Combine(dir, name + ".dll"));
-
+            var assembly = Assembly.LoadFrom(Path.Combine(dir, name + ".dll"));
```

Comment:
The absolutely best thing to do would be to unit-test this, but I can see how creating a test-case for this is going to be fairly involved.

As a second best option, if the use of `LoadFrom()` is critical for correct operations., maybe we should add a comment saying that much wherever we use it?

Reply:
I would also opt for the second option. I'll write up a comment referring to https://docs.microsoft.com/en-us/archive/blogs/suzcook/loadfile-vs-loadfrom as it explains the situation rather well. There is a third option of binding to the AssemblyResolve event but that overcomplicates things for the project's scope.